samples/hello_classification.js

```
@@ -0,0 +1,95 @@
+const cv = require('opencv4nodejs');
+const {Core, CNNNetwork} = require('bindings')('InferenceEngineJS');
+
+const patToModel = `${process.env.MODELS_PATH}/classification/inception_v3/inception_v3.`;
```

should we throw error if `MODELS_PATH` is not set?